docs/platform-integration/sensors/index.md

```diff
@@ -0,0 +1,245 @@
+
+- `Game`\
+Rate suitable for games (not guaranteed to return on UI thread).
+
+- `Default`\
+Default rate suitable for screen orientation changes.
+
+- `UI`\
+Rate suitable for general user interface.
+
+> [!WARNING]
+> Monitoring too many sensors at once may affect the rate sensor data is returned to your app.
+
+### Sensor event handlers
+
+Event handlers added to sensors with either the `Game` or `Fastest` speeds **aren't** guaranteed to run on the UI thread. If the event handler needs to access user-interface elements, use the [`MainThread.BeginInvokeOnMainThread`](../appmodel/main-thread.md) method to run that code on the UI thread.
+
+## Accelerometer
+
+The accelerometer sensor measures the acceleration of the device along its three axes. The data reported by the sensor represents how the user is moving the device.
+
+To start monitoring the accelerometer sensor, call the `IAccelerometer.Start` method. .NET MAUI sends accelerometer data changes to your app by raising the `IAccelerometer.ReadingChanged` event. Use the `IAccelerometer.Stop` method to stop monitoring the sensor. You can detect the monitoring state of the accelerometer with the `IAccelerometer.IsMonitoring` property, which will be `true` if the accelerometer was started and is currently being monitored.
+
+The following code example demonstrates monitoring the accelerometer for changes. The `Accelerometer.Default` instance would be passed to the `ToggleAccelerometer` method:
+
+:::code language="csharp" source="../snippets/shared_1/SensorsPage.xaml.cs" id="toggle_accelerometer":::
+
+Accelerometer readings are reported back in **G**. A **G** is a unit of gravitation force equal to the gravity exerted by the earth's gravitational field $(9.81 m/s^2)$.
+
+The coordinate-system is defined relative to the screen of the device in its default orientation. The axes aren't swapped when the device's screen orientation changes.
+
+The **X** axis is horizontal and points to the right, the **Y** axis is vertical and points up and the **Z** axis points towards the outside of the front face of the screen. In this system, coordinates behind the screen have negative **Z** values.
+
+Examples:
+
+- When the device lies flat on a table and is pushed on its left side toward the right, the **X** acceleration value is positive.
+
+- When the device lies flat on a table, the acceleration value is +1.00 G or $(+9.81 m/s^2)$, which correspond to the acceleration of the device $(0 m/s^2)$ minus the force of gravity $(-9.81 m/s^2)$ and normalized as in G.
+
+<!-- TODO: Why are A and G mentioned here as if they're XYZ properties from the data? -->
+- When the device lies flat on a table and is pushed toward the sky with an acceleration of **A** $m/s^2$, the acceleration value is equal to $A+9.81$ which corresponds to the acceleration of the device $(+A m/s^2)$ minus the force of gravity $(-9.81 m/s^2)$ and normalized in **G**.
+
+### Platform-specific information (Accelerometer)
```

Comment:
Delete section?

Reply:
@davidbritch I don't feel strongly about having these. I feel that because some of the sensors have this info, and are in the same singluar doc, I didn't want readers to think something was missing from some of the sensor info.

If you feel strongly about deleting them, let me know and I'll do it.

Reply:
No strong opinions on this.